src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
The creation of 'FigureSupplier' instance is done correctly. Make sure that 'FigureSupplier' does not generate random values inside figure classes as per the checklist.